data/102/084/283/102084283.geojson

```diff
@@ -2,7 +2,7 @@
   "id": 102084283,
   "type": "Feature",
   "properties": {
-    "edtf:cessation":"uuuu",
+    "edtf:cessation":"2021-12-31",
```

Comment:
This and a few others should get an update to the `wof:lastmodified` date field.

Reply:
Fixed by re-exportifying everything!